vars/manifest.yml

```diff
@@ -2,37 +2,46 @@
 upstream_repos:
   openstack-helm-infra:
     src: https://git.openstack.org/openstack/openstack-helm-infra
-    version: master
+    # Commit on April 24, 2019
+    version: 0ed4f0de5ed7b0eb7bb6cb6f819ad48e3fd71a23
     dest_subdir: openstack
   openstack-helm:
     src: https://git.openstack.org/openstack/openstack-helm
-    version: master
+    # Commit on April 28,2019
+    version: 4b4745f1cda13d2d1c7b92b474135ebfb7ee689c
```

Comment:
do we need to pin openstack helm here? don't think it is used by product deployment.

Reply:
Reason I did was for consistency sake as it can be confusing for non-team developer member to understand nuances of the reasoning. 
Don't we need to pin these repos for OSH deployment mechanism as well. I was thinking this may be the time.